web_monitoring/diffing_server.py

```diff
@@ -136,8 +136,17 @@ def get(self, differ):
                     body = f.read()
                     responses[param] = MockResponse(url, body, headers)
         # Now fetch any nonlocal URLs.
+        # Pass request header defined by URL param pass_header=HEADER_NAME
+        # to nonlocal URLs. Useful for passing data like cookie headers.
         to_fetch = {k: v for k, v in urls.items() if k not in responses}
-        fetched = yield [client.fetch(url, raise_error=False)
+        headers = {}
+        header_key = query_params.get('pass_header')
```

Comment:
I’m imagining we might want to support multiple headers here. Not sure whether the best API for that is:

```
# multiple args
GET /some_diff?pass_header=Authorization&pass_header=Accept

# or comma separated
GET /some_diff?pass_header=Authorization,Accept

# with [] to indicate a list
GET /some_diff?pass_header[]=Authorization&pass_header[]=Accept
```

I’ve seen all of those as common styles. I’d be fine with whichever you think makes the most sense.